cmd/otelcol/config/collector/agent_config_linux.yaml

```
@@ -85,7 +82,7 @@ service:
     traces:
       receivers: [jaeger, opencensus, otlp, sapm, zipkin]
       processors: [memory_limiter, batch]
-      exporters: [sapm, signalfx_correlation]
+      exporters: [sapm]
```

signalfx exporter can go in traces pipeline now for correlation:

```suggestion
      exporters: [sapm, signalfx]
```